bin/anthology/index.py

```diff
@@ -354,30 +354,6 @@ def similar(self):
             )
         return self._similar
 
-    def verify(self):
```

Comment:
Can you comment why we no longer need to verify?

Reply:
https://github.com/acl-org/acl-anthology/pull/2798#issuecomment-1723232465